samples/contractupdate/src/WalletConnectionButton.tsx

```diff
@@ -17,8 +18,8 @@ export function WalletConnectionButton({
             {activeConnector && !activeConnection && (
                 <Button type="button" onClick={connectActive} disabled={isConnecting}>
                     {isConnecting && 'Connecting...'}
-                    {!isConnecting && activeConnectorType === 'BrowserWallet' && 'Connect Browser Wallet'}
-                    {!isConnecting && activeConnectorType === 'WalletConnect' && 'Connect Mobile Wallet'}
```

Comment:
Seems weird to me that this component is dependant on `config` and the concrete `ConnectorType` implementations. Could just add `children` to props and define the text of the button there instead? I know this is an example, but I thought it'd make better sense anyway.

Reply:
Not sure I agree it's an improvement, but ok it's done :)